opentreemap/treemap/tests/test_audit.py

```diff
@@ -385,376 +376,6 @@ def test_approving_edits_on_deleted_obj_doesnt_fail(self):
             width_audit, self.commander_user, approved=False)
 
 
-class PendingTest(OTMTestCase):
```

Comment:
I like skipping them with a message that explains they are not compatible with contrib hstore based UDF system.